Source/Core/DiscIO/VolumeGC.cpp

```diff
@@ -208,8 +208,10 @@ void CVolumeGC::LoadBannerFile() const
 
   GCBanner banner_file;
   std::unique_ptr<IFileSystem> file_system(CreateFileSystem(this, PARTITION_NONE));
-  size_t file_size = (size_t)file_system->GetFileSize("opening.bnr");
+  if (!file_system)
+    return;
 
+  size_t file_size = (size_t)file_system->GetFileSize("opening.bnr");
```

Comment:
Isn't `size_t` [implementation dependant](http://en.cppreference.com/w/cpp/types/size_t) on whether its a 32-bit or 64-bit value? Just something to consider, altho it probably doesn't make much of a difference in practise.